### PR TITLE
Chore/static pages

### PR DESCRIPTION
**Pull request by fr0gz**
created new app for managing static pages

### CODE REVIEW

File: cmpirque/static_pages/admin.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
+# Register your models here.
```
Comment (chivke) [inline cmpirque/static_pages/admin.py:3]: Are we going to use django admin?

File: cmpirque/static_pages/tests.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```
Comment (chivke) [inline cmpirque/static_pages/tests.py:3]: This is not correct, the tests must be in a folder with that name, like `cmpirque/pages/tests/` It is done to modularize the tests by parts: `test_models`, `test_views`, etc.

File: cmpirque/static_pages/views.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+from django.shortcuts import render
+
+# Create your views here.
```
Comment (chivke) [inline cmpirque/static_pages/views.py:3]: The views should be a folder too

File: config/settings/base.py [hunk 4]
```diff
@@ -183,7 +183,7 @@
 
 # Slack's app token
 
-SLACK_BOT_TOKEN = env('SLACK_BOT_TOKEN')
+#SLACK_BOT_TOKEN = env('SLACK_BOT_TOKEN')
```
Comment (chivke) [inline config/settings/base.py:186]: Delete it.

**Review by chivke — CHANGES_REQUESTED** [review]
Look the comments! I don't like the name `static_pages`, I think `pages` name it's enough. Tell me what your think.

File: cmpirque/static_pages/admin.py (hunk 1 above)
Comment (fr0gz) [reply cmpirque/static_pages/admin.py:3]: I will delete it

File: cmpirque/static_pages/views.py (hunk 3 above)
Comment (fr0gz) [reply cmpirque/static_pages/views.py:3]: I will change it to cmpirque/pages/views

**Review by chivke — APPROVED** [review]

**Pull request merged**